### ISSUE TITLE
Logging for multiple clients in netopeer

### ISSUE THREAD

**Issue by bvirmani2022** [CESNET/netopeer2#1476]
Hi, Currently the logs in netopeer can be seen in the syslogs as
---- netopeer-server ----------

Is it possible to send logs to the syslogs for multiple clients as:
-----netopeer-server1---------
-----netopeer-server2--------

From where can I differentiate and add appropriate checks for sending logs to syslogs in the above format

**Comment by michalvasko** [comment]
Not sure what you mean by "clients" but I can only think of NETCONF clients. Each has their own session, which is printed at the beginning of the message if applicable. You should be able to filter based on that.

**Comment by bvirmani2022** [comment]
Yes, I m talking about the netconf clients only.
But I want the syslog based on the netconf client user logged in and not with the session information of the client.

Is that possible?

**Comment by michalvasko** [comment]
No, it is not possible to change logging in any way. But like I said, you can use sessions, every NETCONF client has its own session.
